providers/implementations/storemgmt/file_store.c

```diff
@@ -221,13 +221,11 @@ static void *file_open(void *provctx, const char *uri)
     if (strncasecmp(uri, "file:", 5) == 0) {
         const char *p = &uri[5];
 
-        if (strncmp(&uri[5], "//", 2) == 0) {
+        if (CHECK_AND_SKIP_PREFIX(p, "//")) {
```

Comment:
Is it worthwhile having case insensitive flavours of these macros?

Reply:
I was wondering about this, too, and just checked this: yes, variant using `strncasecmp()` would be worth to have.
I'm going to add them to this PR.

Reply:
Or better do this in a separate PR, since this one is already approved?

Reply:
Maybe wait for the OTC decision -- if this waits until after 3.0, they can go here.

Reply:
> Maybe wait for the OTC decision -- if this waits until after 3.0, they can go here.

When is this OTC decision done?